ansible/group_vars/all/config.yml

```
@@ -166,6 +166,33 @@ configuration_all:
     server: true
     client: false
     nuxtclient: false
+  # Identity Management
+  FEATURE_IDENTITY_MANAGEMENT_ENABLED:
+    value: 'false'
+    server: true
+    client: false
+    nuxtclient: false
+  IDENTITY_MANAGEMENT__TENANT:
+    value: 'dBildungscloud'
+    server: true
+    client: false
+    nuxtclient: false
+  IDENTITY_MANAGEMENT__ADMIN_CLIENTID:
+    value: 'admin-cli'
+    server: true
+    client: false
+    nuxtclient: false
+  # TODO Where and how to pass credentials to schulcloud-server
+  IDENTITY_MANAGEMENT__ADMIN_USER:
+    value: 'TBA'
+    server: true
+    client: false
+    nuxtclient: false
+  IDENTITY_MANAGEMENT__ADMIN_PASSWORD:
+    value: 'TBA'
```

Manuel Linter says use " please
```suggestion
    value: "TBA"
```

---

done